hack/test-cmd.sh

```diff
@@ -301,7 +301,7 @@ os::cmd::expect_success 'oc logout'
 os::cmd::expect_failure_and_text 'oc get pods' '"system:anonymous" cannot list pods'
 
 # make sure we handle invalid config file destination
-os::cmd::expect_failure_and_text "oc login '${KUBERNETES_MASTER}' -u test -p test --config=/src --insecure-skip-tls-verify" 'KUBECONFIG is set to a file that cannot be created or modified'
+os::cmd::expect_failure_and_text "oc login '${KUBERNETES_MASTER}' -u test -p test --config=/tmp --insecure-skip-tls-verify" 'Error loading config file'
```

Comment:
This looks to me like it should be a separate PR, unless there's something related to the role with this change that I'm somehow missing.